Formula/libreadline-java.rb

```diff
@@ -44,6 +30,7 @@ def install
     # adjust gnu install parameters to bsd install
     inreplace "Makefile" do |s|
       s.change_make_var! "PREFIX", prefix
+      s.change_make_var! "JAVAC_VERSION", Formula["openjdk"].version
```

Comment:
Command to get Java version fails due to Homebrew's `_JAVA_OPTIONS`. Just manually set version here rather than deleting `_JAVA_OPTIONS`.